### ISSUE TITLE
Automating script for magazine download

### ISSUE THREAD

**Issue by steve1977** [skyme5/magzdb#100]
Really love the script. A few ideas how this can be automated:

* Have a folder where I can place magazine ID (as file without extension). This can be used to run the script automatically for all magazine IDs placed in this folder

* Rename and tag files after download

* Check if magazine issue already downloaded and not download again if exists

**Comment by skyme5** [comment]
@steve1977 I agree with adding support for some automation. Will add this to todo.

**Comment by steve1977** [comment]
This would be really cool (together with renaming and an unraid docker). Happy to help testing.
